src/schoology-extractor/data/.gitignore

```diff
@@ -0,0 +1,4 @@
+# Ignore everything in this directory
+*
+# Except this file - it's the directory placeholder
+!.gitignore
```

Comment:
Standard convention is to have a `.gitkeep` file in this directory. The root level `.gitignore` should already be ignoring `*.csv`.

Reply:
But this file is fine too I suppose 😃 .

Reply:
I made the same comment in a past PR from Brad. I think it is fine to have a .gitignore ignoring all the contained files, because of I think we should not commit anything contained in this directory.

Reply:
I really like this style as opposed to the .gitkeep convention.  The ignore everything prevents someone from ever accidentally placing a new (non-ignored elsewhere) file type in there and accidentally committing.  Also, it doesn't rely on convention.

Reply:
Good case for this approach instead of `.gitkeep`.